<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionPointsGUI</class>
 <widget class="QWidget" name="SectionPointsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLabelRichName" name="nameLabel">
     <property name="text">
      <string>Denominazione</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>264</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="materialLabel">
     <property name="text">
      <string>Materiale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabelRichName" name="pWLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
p&lt;span style=&quot; vertical-align:sub;&quot;&gt;w&lt;/span&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabelUnitMeasure" name="pWUMLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[N/m]
&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEditValue" name="pWLEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBoxReadOnly" name="pWCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pointsTab">
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="pointsTableView"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="addPointPButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="removePointPButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="geometricDataTab">
      <attribute name="title">
       <string>Geometria Aree</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="geometricDataScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="geometricDataScrollContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>80</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="transformationTab">
      <attribute name="title">
       <string>Trasformazioni</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="materialComboBox"/>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEditValue" name="nameLEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelRichName</class>
   <extends>QLabel</extends>
   <header>qlabelrichname.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditValue</class>
   <extends>QLineEdit</extends>
   <header>qlineeditvalue.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelUnitMeasure</class>
   <extends>QLabel</extends>
   <header>qlabelunitmeasure.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxReadOnly</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxreadonly.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
